sdk/cmake/CMakeMacros.cmake

```diff
@@ -908,6 +908,21 @@ function(create_registry_hives)
 
 endfunction()
 
+function(add_driver_inf _module)
```

Comment:
Would be nice if this function were designed such that `media/inf/CMakeLists.txt` can use it, too.